Gemfile.lock

```diff
@@ -63,21 +63,21 @@ GEM
     browser_details (0.0.6)
```

Comment:
Can you delete this `Gemfile.lock` file and run `bundle install` again? We don't want to update these gems in this PR too.